.gitignore

```diff
@@ -32,3 +32,6 @@ resotocore/tools/Assets
 out/
 *.iml
 **/.hypothesis/
+plugins/api_sandboxes/*.ipynb
+!plugins/api_sandboxes/*.example.ipynb
```

Comment:
Let's remove this line and the corresponding example file.
Create a `README.md` file that explains the reason for this folder, ala:

```
All files placed in this directory are ignored by git.
You can use this directory to put files and directories for sandbox testing.
```